<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns='http://www.w3.org/1999/xhtml' xml:lang='en' lang='en'>
<head>
<title>openembedded-core - OpenEmbedded Core layer</title>
<meta name='generator' content='cgit v0.9.2-21-gd62e'/>
<meta name='robots' content='index, nofollow'/>
<link rel='stylesheet' type='text/css' href='/cgit.css'/>
<link rel='shortcut icon' href='/favicon.ico'/>
<link rel='alternate' title='Atom feed' href='http://cgit.openembedded.org/openembedded-core/atom/meta/recipes-devtools/rpm/rpm-4.11.2/support-suggests-tag.patch?h=master' type='application/atom+xml'/>
</head>
<body>
<div id='cgit'><table id='header'>
<tr>
<td class='logo' rowspan='2'><a href='/'><img src='/oe.png' alt='cgit logo'/></a></td>
<td class='main'><a href='/'>index</a> : <a title='openembedded-core' href='/openembedded-core/'>openembedded-core</a></td><td class='form'><form method='get' action=''>
<select name='h' onchange='this.form.submit();'>
<option value='2011-1'>2011-1</option>
<option value='daisy'>daisy</option>
<option value='danny'>danny</option>
<option value='denzil'>denzil</option>
<option value='dizzy'>dizzy</option>
<option value='dora'>dora</option>
<option value='dylan'>dylan</option>
<option value='master' selected='selected'>master</option>
<option value='master-next'>master-next</option>
</select> <input type='submit' name='' value='switch'/></form></td></tr>
<tr><td class='sub'>OpenEmbedded Core layer</td><td class='sub right'>OpenEmbedded</td></tr></table>
<table class='tabs'><tr><td>
<a href='/openembedded-core/'>summary</a><a href='/openembedded-core/refs/'>refs</a><a href='/openembedded-core/log/meta/recipes-devtools/rpm/rpm-4.11.2/support-suggests-tag.patch'>log</a><a class='active' href='/openembedded-core/tree/meta/recipes-devtools/rpm/rpm-4.11.2/support-suggests-tag.patch'>tree</a><a href='/openembedded-core/commit/meta/recipes-devtools/rpm/rpm-4.11.2/support-suggests-tag.patch'>commit</a><a href='/openembedded-core/diff/meta/recipes-devtools/rpm/rpm-4.11.2/support-suggests-tag.patch'>diff</a><a href='/openembedded-core/about/'>about</a></td><td class='form'><form class='right' method='get' action='/openembedded-core/log/meta/recipes-devtools/rpm/rpm-4.11.2/support-suggests-tag.patch'>
<select name='qt'>
<option value='grep'>log msg</option>
<option value='author'>author</option>
<option value='committer'>committer</option>
<option value='range'>range</option>
</select>
<input class='txt' type='text' size='10' name='q' value=''/>
<input type='submit' value='search'/>
</form>
</td></tr></table>
<div class='path'>path: <a href='/openembedded-core/tree/'>root</a>/<a href='/openembedded-core/tree/meta'>meta</a>/<a href='/openembedded-core/tree/meta/recipes-devtools'>recipes-devtools</a>/<a href='/openembedded-core/tree/meta/recipes-devtools/rpm'>rpm</a>/<a href='/openembedded-core/tree/meta/recipes-devtools/rpm/rpm-4.11.2'>rpm-4.11.2</a>/<a href='/openembedded-core/tree/meta/recipes-devtools/rpm/rpm-4.11.2/support-suggests-tag.patch'>support-suggests-tag.patch</a></div><div class='content'>blob: 3da608573aae76d0698032aeef035fe732d0ad3d (<a href='/openembedded-core/plain/meta/recipes-devtools/rpm/rpm-4.11.2/support-suggests-tag.patch'>plain</a>)
<table summary='blob content' class='blob'>
<tr><td class='linenumbers'><pre><a class='no' id='n1' name='n1' href='#n1'>1</a>
<a class='no' id='n2' name='n2' href='#n2'>2</a>
<a class='no' id='n3' name='n3' href='#n3'>3</a>
<a class='no' id='n4' name='n4' href='#n4'>4</a>
<a class='no' id='n5' name='n5' href='#n5'>5</a>
<a class='no' id='n6' name='n6' href='#n6'>6</a>
<a class='no' id='n7' name='n7' href='#n7'>7</a>
<a class='no' id='n8' name='n8' href='#n8'>8</a>
<a class='no' id='n9' name='n9' href='#n9'>9</a>
<a class='no' id='n10' name='n10' href='#n10'>10</a>
<a class='no' id='n11' name='n11' href='#n11'>11</a>
<a class='no' id='n12' name='n12' href='#n12'>12</a>
<a class='no' id='n13' name='n13' href='#n13'>13</a>
<a class='no' id='n14' name='n14' href='#n14'>14</a>
<a class='no' id='n15' name='n15' href='#n15'>15</a>
<a class='no' id='n16' name='n16' href='#n16'>16</a>
<a class='no' id='n17' name='n17' href='#n17'>17</a>
<a class='no' id='n18' name='n18' href='#n18'>18</a>
<a class='no' id='n19' name='n19' href='#n19'>19</a>
<a class='no' id='n20' name='n20' href='#n20'>20</a>
<a class='no' id='n21' name='n21' href='#n21'>21</a>
<a class='no' id='n22' name='n22' href='#n22'>22</a>
<a class='no' id='n23' name='n23' href='#n23'>23</a>
<a class='no' id='n24' name='n24' href='#n24'>24</a>
<a class='no' id='n25' name='n25' href='#n25'>25</a>
<a class='no' id='n26' name='n26' href='#n26'>26</a>
<a class='no' id='n27' name='n27' href='#n27'>27</a>
<a class='no' id='n28' name='n28' href='#n28'>28</a>
<a class='no' id='n29' name='n29' href='#n29'>29</a>
<a class='no' id='n30' name='n30' href='#n30'>30</a>
<a class='no' id='n31' name='n31' href='#n31'>31</a>
<a class='no' id='n32' name='n32' href='#n32'>32</a>
<a class='no' id='n33' name='n33' href='#n33'>33</a>
<a class='no' id='n34' name='n34' href='#n34'>34</a>
<a class='no' id='n35' name='n35' href='#n35'>35</a>
<a class='no' id='n36' name='n36' href='#n36'>36</a>
<a class='no' id='n37' name='n37' href='#n37'>37</a>
<a class='no' id='n38' name='n38' href='#n38'>38</a>
<a class='no' id='n39' name='n39' href='#n39'>39</a>
<a class='no' id='n40' name='n40' href='#n40'>40</a>
<a class='no' id='n41' name='n41' href='#n41'>41</a>
<a class='no' id='n42' name='n42' href='#n42'>42</a>
<a class='no' id='n43' name='n43' href='#n43'>43</a>
<a class='no' id='n44' name='n44' href='#n44'>44</a>
<a class='no' id='n45' name='n45' href='#n45'>45</a>
<a class='no' id='n46' name='n46' href='#n46'>46</a>
<a class='no' id='n47' name='n47' href='#n47'>47</a>
<a class='no' id='n48' name='n48' href='#n48'>48</a>
<a class='no' id='n49' name='n49' href='#n49'>49</a>
<a class='no' id='n50' name='n50' href='#n50'>50</a>
<a class='no' id='n51' name='n51' href='#n51'>51</a>
<a class='no' id='n52' name='n52' href='#n52'>52</a>
<a class='no' id='n53' name='n53' href='#n53'>53</a>
<a class='no' id='n54' name='n54' href='#n54'>54</a>
<a class='no' id='n55' name='n55' href='#n55'>55</a>
<a class='no' id='n56' name='n56' href='#n56'>56</a>
<a class='no' id='n57' name='n57' href='#n57'>57</a>
<a class='no' id='n58' name='n58' href='#n58'>58</a>
<a class='no' id='n59' name='n59' href='#n59'>59</a>
<a class='no' id='n60' name='n60' href='#n60'>60</a>
<a class='no' id='n61' name='n61' href='#n61'>61</a>
<a class='no' id='n62' name='n62' href='#n62'>62</a>
<a class='no' id='n63' name='n63' href='#n63'>63</a>
<a class='no' id='n64' name='n64' href='#n64'>64</a>
<a class='no' id='n65' name='n65' href='#n65'>65</a>
<a class='no' id='n66' name='n66' href='#n66'>66</a>
<a class='no' id='n67' name='n67' href='#n67'>67</a>
<a class='no' id='n68' name='n68' href='#n68'>68</a>
<a class='no' id='n69' name='n69' href='#n69'>69</a>
<a class='no' id='n70' name='n70' href='#n70'>70</a>
<a class='no' id='n71' name='n71' href='#n71'>71</a>
<a class='no' id='n72' name='n72' href='#n72'>72</a>
<a class='no' id='n73' name='n73' href='#n73'>73</a>
<a class='no' id='n74' name='n74' href='#n74'>74</a>
<a class='no' id='n75' name='n75' href='#n75'>75</a>
<a class='no' id='n76' name='n76' href='#n76'>76</a>
<a class='no' id='n77' name='n77' href='#n77'>77</a>
<a class='no' id='n78' name='n78' href='#n78'>78</a>
<a class='no' id='n79' name='n79' href='#n79'>79</a>
<a class='no' id='n80' name='n80' href='#n80'>80</a>
<a class='no' id='n81' name='n81' href='#n81'>81</a>
<a class='no' id='n82' name='n82' href='#n82'>82</a>
<a class='no' id='n83' name='n83' href='#n83'>83</a>
<a class='no' id='n84' name='n84' href='#n84'>84</a>
<a class='no' id='n85' name='n85' href='#n85'>85</a>
<a class='no' id='n86' name='n86' href='#n86'>86</a>
<a class='no' id='n87' name='n87' href='#n87'>87</a>
<a class='no' id='n88' name='n88' href='#n88'>88</a>
<a class='no' id='n89' name='n89' href='#n89'>89</a>
<a class='no' id='n90' name='n90' href='#n90'>90</a>
<a class='no' id='n91' name='n91' href='#n91'>91</a>
<a class='no' id='n92' name='n92' href='#n92'>92</a>
<a class='no' id='n93' name='n93' href='#n93'>93</a>
<a class='no' id='n94' name='n94' href='#n94'>94</a>
<a class='no' id='n95' name='n95' href='#n95'>95</a>
<a class='no' id='n96' name='n96' href='#n96'>96</a>
<a class='no' id='n97' name='n97' href='#n97'>97</a>
<a class='no' id='n98' name='n98' href='#n98'>98</a>
<a class='no' id='n99' name='n99' href='#n99'>99</a>
<a class='no' id='n100' name='n100' href='#n100'>100</a>
<a class='no' id='n101' name='n101' href='#n101'>101</a>
<a class='no' id='n102' name='n102' href='#n102'>102</a>
<a class='no' id='n103' name='n103' href='#n103'>103</a>
<a class='no' id='n104' name='n104' href='#n104'>104</a>
<a class='no' id='n105' name='n105' href='#n105'>105</a>
<a class='no' id='n106' name='n106' href='#n106'>106</a>
<a class='no' id='n107' name='n107' href='#n107'>107</a>
<a class='no' id='n108' name='n108' href='#n108'>108</a>
<a class='no' id='n109' name='n109' href='#n109'>109</a>
<a class='no' id='n110' name='n110' href='#n110'>110</a>
<a class='no' id='n111' name='n111' href='#n111'>111</a>
<a class='no' id='n112' name='n112' href='#n112'>112</a>
<a class='no' id='n113' name='n113' href='#n113'>113</a>
<a class='no' id='n114' name='n114' href='#n114'>114</a>
<a class='no' id='n115' name='n115' href='#n115'>115</a>
<a class='no' id='n116' name='n116' href='#n116'>116</a>
<a class='no' id='n117' name='n117' href='#n117'>117</a>
<a class='no' id='n118' name='n118' href='#n118'>118</a>
<a class='no' id='n119' name='n119' href='#n119'>119</a>
<a class='no' id='n120' name='n120' href='#n120'>120</a>
<a class='no' id='n121' name='n121' href='#n121'>121</a>
<a class='no' id='n122' name='n122' href='#n122'>122</a>
<a class='no' id='n123' name='n123' href='#n123'>123</a>
<a class='no' id='n124' name='n124' href='#n124'>124</a>
<a class='no' id='n125' name='n125' href='#n125'>125</a>
<a class='no' id='n126' name='n126' href='#n126'>126</a>
<a class='no' id='n127' name='n127' href='#n127'>127</a>
<a class='no' id='n128' name='n128' href='#n128'>128</a>
<a class='no' id='n129' name='n129' href='#n129'>129</a>
<a class='no' id='n130' name='n130' href='#n130'>130</a>
<a class='no' id='n131' name='n131' href='#n131'>131</a>
<a class='no' id='n132' name='n132' href='#n132'>132</a>
<a class='no' id='n133' name='n133' href='#n133'>133</a>
<a class='no' id='n134' name='n134' href='#n134'>134</a>
<a class='no' id='n135' name='n135' href='#n135'>135</a>
<a class='no' id='n136' name='n136' href='#n136'>136</a>
<a class='no' id='n137' name='n137' href='#n137'>137</a>
<a class='no' id='n138' name='n138' href='#n138'>138</a>
<a class='no' id='n139' name='n139' href='#n139'>139</a>
<a class='no' id='n140' name='n140' href='#n140'>140</a>
<a class='no' id='n141' name='n141' href='#n141'>141</a>
<a class='no' id='n142' name='n142' href='#n142'>142</a>
<a class='no' id='n143' name='n143' href='#n143'>143</a>
<a class='no' id='n144' name='n144' href='#n144'>144</a>
<a class='no' id='n145' name='n145' href='#n145'>145</a>
<a class='no' id='n146' name='n146' href='#n146'>146</a>
<a class='no' id='n147' name='n147' href='#n147'>147</a>
<a class='no' id='n148' name='n148' href='#n148'>148</a>
<a class='no' id='n149' name='n149' href='#n149'>149</a>
<a class='no' id='n150' name='n150' href='#n150'>150</a>
<a class='no' id='n151' name='n151' href='#n151'>151</a>
<a class='no' id='n152' name='n152' href='#n152'>152</a>
<a class='no' id='n153' name='n153' href='#n153'>153</a>
<a class='no' id='n154' name='n154' href='#n154'>154</a>
<a class='no' id='n155' name='n155' href='#n155'>155</a>
<a class='no' id='n156' name='n156' href='#n156'>156</a>
<a class='no' id='n157' name='n157' href='#n157'>157</a>
<a class='no' id='n158' name='n158' href='#n158'>158</a>
<a class='no' id='n159' name='n159' href='#n159'>159</a>
<a class='no' id='n160' name='n160' href='#n160'>160</a>
<a class='no' id='n161' name='n161' href='#n161'>161</a>
<a class='no' id='n162' name='n162' href='#n162'>162</a>
<a class='no' id='n163' name='n163' href='#n163'>163</a>
<a class='no' id='n164' name='n164' href='#n164'>164</a>
<a class='no' id='n165' name='n165' href='#n165'>165</a>
<a class='no' id='n166' name='n166' href='#n166'>166</a>
<a class='no' id='n167' name='n167' href='#n167'>167</a>
<a class='no' id='n168' name='n168' href='#n168'>168</a>
<a class='no' id='n169' name='n169' href='#n169'>169</a>
<a class='no' id='n170' name='n170' href='#n170'>170</a>
<a class='no' id='n171' name='n171' href='#n171'>171</a>
<a class='no' id='n172' name='n172' href='#n172'>172</a>
<a class='no' id='n173' name='n173' href='#n173'>173</a>
<a class='no' id='n174' name='n174' href='#n174'>174</a>
<a class='no' id='n175' name='n175' href='#n175'>175</a>
<a class='no' id='n176' name='n176' href='#n176'>176</a>
<a class='no' id='n177' name='n177' href='#n177'>177</a>
<a class='no' id='n178' name='n178' href='#n178'>178</a>
<a class='no' id='n179' name='n179' href='#n179'>179</a>
<a class='no' id='n180' name='n180' href='#n180'>180</a>
<a class='no' id='n181' name='n181' href='#n181'>181</a>
<a class='no' id='n182' name='n182' href='#n182'>182</a>
<a class='no' id='n183' name='n183' href='#n183'>183</a>
<a class='no' id='n184' name='n184' href='#n184'>184</a>
<a class='no' id='n185' name='n185' href='#n185'>185</a>
<a class='no' id='n186' name='n186' href='#n186'>186</a>
<a class='no' id='n187' name='n187' href='#n187'>187</a>
<a class='no' id='n188' name='n188' href='#n188'>188</a>
<a class='no' id='n189' name='n189' href='#n189'>189</a>
<a class='no' id='n190' name='n190' href='#n190'>190</a>
<a class='no' id='n191' name='n191' href='#n191'>191</a>
<a class='no' id='n192' name='n192' href='#n192'>192</a>
<a class='no' id='n193' name='n193' href='#n193'>193</a>
<a class='no' id='n194' name='n194' href='#n194'>194</a>
<a class='no' id='n195' name='n195' href='#n195'>195</a>
<a class='no' id='n196' name='n196' href='#n196'>196</a>
<a class='no' id='n197' name='n197' href='#n197'>197</a>
<a class='no' id='n198' name='n198' href='#n198'>198</a>
<a class='no' id='n199' name='n199' href='#n199'>199</a>
<a class='no' id='n200' name='n200' href='#n200'>200</a>
<a class='no' id='n201' name='n201' href='#n201'>201</a>
<a class='no' id='n202' name='n202' href='#n202'>202</a>
<a class='no' id='n203' name='n203' href='#n203'>203</a>
<a class='no' id='n204' name='n204' href='#n204'>204</a>
<a class='no' id='n205' name='n205' href='#n205'>205</a>
<a class='no' id='n206' name='n206' href='#n206'>206</a>
<a class='no' id='n207' name='n207' href='#n207'>207</a>
<a class='no' id='n208' name='n208' href='#n208'>208</a>
<a class='no' id='n209' name='n209' href='#n209'>209</a>
<a class='no' id='n210' name='n210' href='#n210'>210</a>
<a class='no' id='n211' name='n211' href='#n211'>211</a>
<a class='no' id='n212' name='n212' href='#n212'>212</a>
<a class='no' id='n213' name='n213' href='#n213'>213</a>
<a class='no' id='n214' name='n214' href='#n214'>214</a>
<a class='no' id='n215' name='n215' href='#n215'>215</a>
<a class='no' id='n216' name='n216' href='#n216'>216</a>
<a class='no' id='n217' name='n217' href='#n217'>217</a>
<a class='no' id='n218' name='n218' href='#n218'>218</a>
<a class='no' id='n219' name='n219' href='#n219'>219</a>
<a class='no' id='n220' name='n220' href='#n220'>220</a>
<a class='no' id='n221' name='n221' href='#n221'>221</a>
<a class='no' id='n222' name='n222' href='#n222'>222</a>
<a class='no' id='n223' name='n223' href='#n223'>223</a>
<a class='no' id='n224' name='n224' href='#n224'>224</a>
<a class='no' id='n225' name='n225' href='#n225'>225</a>
<a class='no' id='n226' name='n226' href='#n226'>226</a>
<a class='no' id='n227' name='n227' href='#n227'>227</a>
<a class='no' id='n228' name='n228' href='#n228'>228</a>
<a class='no' id='n229' name='n229' href='#n229'>229</a>
<a class='no' id='n230' name='n230' href='#n230'>230</a>
<a class='no' id='n231' name='n231' href='#n231'>231</a>
<a class='no' id='n232' name='n232' href='#n232'>232</a>
<a class='no' id='n233' name='n233' href='#n233'>233</a>
<a class='no' id='n234' name='n234' href='#n234'>234</a>
<a class='no' id='n235' name='n235' href='#n235'>235</a>
<a class='no' id='n236' name='n236' href='#n236'>236</a>
<a class='no' id='n237' name='n237' href='#n237'>237</a>
<a class='no' id='n238' name='n238' href='#n238'>238</a>
<a class='no' id='n239' name='n239' href='#n239'>239</a>
<a class='no' id='n240' name='n240' href='#n240'>240</a>
<a class='no' id='n241' name='n241' href='#n241'>241</a>
<a class='no' id='n242' name='n242' href='#n242'>242</a>
<a class='no' id='n243' name='n243' href='#n243'>243</a>
<a class='no' id='n244' name='n244' href='#n244'>244</a>
<a class='no' id='n245' name='n245' href='#n245'>245</a>
<a class='no' id='n246' name='n246' href='#n246'>246</a>
<a class='no' id='n247' name='n247' href='#n247'>247</a>
<a class='no' id='n248' name='n248' href='#n248'>248</a>
<a class='no' id='n249' name='n249' href='#n249'>249</a>
<a class='no' id='n250' name='n250' href='#n250'>250</a>
<a class='no' id='n251' name='n251' href='#n251'>251</a>
<a class='no' id='n252' name='n252' href='#n252'>252</a>
<a class='no' id='n253' name='n253' href='#n253'>253</a>
<a class='no' id='n254' name='n254' href='#n254'>254</a>
<a class='no' id='n255' name='n255' href='#n255'>255</a>
<a class='no' id='n256' name='n256' href='#n256'>256</a>
<a class='no' id='n257' name='n257' href='#n257'>257</a>
<a class='no' id='n258' name='n258' href='#n258'>258</a>
<a class='no' id='n259' name='n259' href='#n259'>259</a>
<a class='no' id='n260' name='n260' href='#n260'>260</a>
<a class='no' id='n261' name='n261' href='#n261'>261</a>
<a class='no' id='n262' name='n262' href='#n262'>262</a>
<a class='no' id='n263' name='n263' href='#n263'>263</a>
<a class='no' id='n264' name='n264' href='#n264'>264</a>
<a class='no' id='n265' name='n265' href='#n265'>265</a>
<a class='no' id='n266' name='n266' href='#n266'>266</a>
<a class='no' id='n267' name='n267' href='#n267'>267</a>
<a class='no' id='n268' name='n268' href='#n268'>268</a>
<a class='no' id='n269' name='n269' href='#n269'>269</a>
<a class='no' id='n270' name='n270' href='#n270'>270</a>
<a class='no' id='n271' name='n271' href='#n271'>271</a>
<a class='no' id='n272' name='n272' href='#n272'>272</a>
<a class='no' id='n273' name='n273' href='#n273'>273</a>
<a class='no' id='n274' name='n274' href='#n274'>274</a>
<a class='no' id='n275' name='n275' href='#n275'>275</a>
<a class='no' id='n276' name='n276' href='#n276'>276</a>
<a class='no' id='n277' name='n277' href='#n277'>277</a>
<a class='no' id='n278' name='n278' href='#n278'>278</a>
<a class='no' id='n279' name='n279' href='#n279'>279</a>
<a class='no' id='n280' name='n280' href='#n280'>280</a>
<a class='no' id='n281' name='n281' href='#n281'>281</a>
<a class='no' id='n282' name='n282' href='#n282'>282</a>
<a class='no' id='n283' name='n283' href='#n283'>283</a>
<a class='no' id='n284' name='n284' href='#n284'>284</a>
<a class='no' id='n285' name='n285' href='#n285'>285</a>
<a class='no' id='n286' name='n286' href='#n286'>286</a>
<a class='no' id='n287' name='n287' href='#n287'>287</a>
<a class='no' id='n288' name='n288' href='#n288'>288</a>
<a class='no' id='n289' name='n289' href='#n289'>289</a>
<a class='no' id='n290' name='n290' href='#n290'>290</a>
<a class='no' id='n291' name='n291' href='#n291'>291</a>
<a class='no' id='n292' name='n292' href='#n292'>292</a>
<a class='no' id='n293' name='n293' href='#n293'>293</a>
<a class='no' id='n294' name='n294' href='#n294'>294</a>
<a class='no' id='n295' name='n295' href='#n295'>295</a>
<a class='no' id='n296' name='n296' href='#n296'>296</a>
<a class='no' id='n297' name='n297' href='#n297'>297</a>
<a class='no' id='n298' name='n298' href='#n298'>298</a>
<a class='no' id='n299' name='n299' href='#n299'>299</a>
<a class='no' id='n300' name='n300' href='#n300'>300</a>
<a class='no' id='n301' name='n301' href='#n301'>301</a>
<a class='no' id='n302' name='n302' href='#n302'>302</a>
<a class='no' id='n303' name='n303' href='#n303'>303</a>
<a class='no' id='n304' name='n304' href='#n304'>304</a>
<a class='no' id='n305' name='n305' href='#n305'>305</a>
<a class='no' id='n306' name='n306' href='#n306'>306</a>
<a class='no' id='n307' name='n307' href='#n307'>307</a>
<a class='no' id='n308' name='n308' href='#n308'>308</a>
<a class='no' id='n309' name='n309' href='#n309'>309</a>
<a class='no' id='n310' name='n310' href='#n310'>310</a>
<a class='no' id='n311' name='n311' href='#n311'>311</a>
<a class='no' id='n312' name='n312' href='#n312'>312</a>
<a class='no' id='n313' name='n313' href='#n313'>313</a>
<a class='no' id='n314' name='n314' href='#n314'>314</a>
<a class='no' id='n315' name='n315' href='#n315'>315</a>
<a class='no' id='n316' name='n316' href='#n316'>316</a>
<a class='no' id='n317' name='n317' href='#n317'>317</a>
<a class='no' id='n318' name='n318' href='#n318'>318</a>
<a class='no' id='n319' name='n319' href='#n319'>319</a>
<a class='no' id='n320' name='n320' href='#n320'>320</a>
<a class='no' id='n321' name='n321' href='#n321'>321</a>
<a class='no' id='n322' name='n322' href='#n322'>322</a>
<a class='no' id='n323' name='n323' href='#n323'>323</a>
<a class='no' id='n324' name='n324' href='#n324'>324</a>
<a class='no' id='n325' name='n325' href='#n325'>325</a>
<a class='no' id='n326' name='n326' href='#n326'>326</a>
<a class='no' id='n327' name='n327' href='#n327'>327</a>
<a class='no' id='n328' name='n328' href='#n328'>328</a>
<a class='no' id='n329' name='n329' href='#n329'>329</a>
<a class='no' id='n330' name='n330' href='#n330'>330</a>
<a class='no' id='n331' name='n331' href='#n331'>331</a>
<a class='no' id='n332' name='n332' href='#n332'>332</a>
<a class='no' id='n333' name='n333' href='#n333'>333</a>
<a class='no' id='n334' name='n334' href='#n334'>334</a>
<a class='no' id='n335' name='n335' href='#n335'>335</a>
<a class='no' id='n336' name='n336' href='#n336'>336</a>
<a class='no' id='n337' name='n337' href='#n337'>337</a>
<a class='no' id='n338' name='n338' href='#n338'>338</a>
<a class='no' id='n339' name='n339' href='#n339'>339</a>
<a class='no' id='n340' name='n340' href='#n340'>340</a>
<a class='no' id='n341' name='n341' href='#n341'>341</a>
<a class='no' id='n342' name='n342' href='#n342'>342</a>
<a class='no' id='n343' name='n343' href='#n343'>343</a>
<a class='no' id='n344' name='n344' href='#n344'>344</a>
<a class='no' id='n345' name='n345' href='#n345'>345</a>
<a class='no' id='n346' name='n346' href='#n346'>346</a>
<a class='no' id='n347' name='n347' href='#n347'>347</a>
<a class='no' id='n348' name='n348' href='#n348'>348</a>
<a class='no' id='n349' name='n349' href='#n349'>349</a>
<a class='no' id='n350' name='n350' href='#n350'>350</a>
<a class='no' id='n351' name='n351' href='#n351'>351</a>
<a class='no' id='n352' name='n352' href='#n352'>352</a>
<a class='no' id='n353' name='n353' href='#n353'>353</a>
<a class='no' id='n354' name='n354' href='#n354'>354</a>
<a class='no' id='n355' name='n355' href='#n355'>355</a>
<a class='no' id='n356' name='n356' href='#n356'>356</a>
<a class='no' id='n357' name='n357' href='#n357'>357</a>
<a class='no' id='n358' name='n358' href='#n358'>358</a>
<a class='no' id='n359' name='n359' href='#n359'>359</a>
<a class='no' id='n360' name='n360' href='#n360'>360</a>
<a class='no' id='n361' name='n361' href='#n361'>361</a>
<a class='no' id='n362' name='n362' href='#n362'>362</a>
<a class='no' id='n363' name='n363' href='#n363'>363</a>
<a class='no' id='n364' name='n364' href='#n364'>364</a>
<a class='no' id='n365' name='n365' href='#n365'>365</a>
<a class='no' id='n366' name='n366' href='#n366'>366</a>
<a class='no' id='n367' name='n367' href='#n367'>367</a>
<a class='no' id='n368' name='n368' href='#n368'>368</a>
<a class='no' id='n369' name='n369' href='#n369'>369</a>
<a class='no' id='n370' name='n370' href='#n370'>370</a>
<a class='no' id='n371' name='n371' href='#n371'>371</a>
<a class='no' id='n372' name='n372' href='#n372'>372</a>
<a class='no' id='n373' name='n373' href='#n373'>373</a>
<a class='no' id='n374' name='n374' href='#n374'>374</a>
<a class='no' id='n375' name='n375' href='#n375'>375</a>
<a class='no' id='n376' name='n376' href='#n376'>376</a>
<a class='no' id='n377' name='n377' href='#n377'>377</a>
<a class='no' id='n378' name='n378' href='#n378'>378</a>
<a class='no' id='n379' name='n379' href='#n379'>379</a>
<a class='no' id='n380' name='n380' href='#n380'>380</a>
<a class='no' id='n381' name='n381' href='#n381'>381</a>
<a class='no' id='n382' name='n382' href='#n382'>382</a>
<a class='no' id='n383' name='n383' href='#n383'>383</a>
<a class='no' id='n384' name='n384' href='#n384'>384</a>
</pre></td>
<td class='lines'><pre><code>
Upstream-Status: Pending

Signed-off-by: Saul Wold &lt;sgw&#64;linux.intel.com&gt;
Signed-off-by: Ronan Le Martret &lt;ronan.lemartret&#64;open.eurogiciel.org&gt;

diff --git a/build/pack.c b/build/pack.c
index b6b2bea..5c3d8df 100644
<span class="hl kwb">--- a/build/pack.c</span>
<span class="hl kwa">+++ b/build/pack.c</span>
&#64;&#64; -273,8 +273,10 &#64;&#64; static rpmTagVal depevrtags[] = {
     RPMTAG_CONFLICTVERSION,
     RPMTAG_ORDERVERSION,
     RPMTAG_TRIGGERVERSION,
<span class="hl kwb">-    RPMTAG_SUGGESTSVERSION,</span>
<span class="hl kwb">-    RPMTAG_ENHANCESVERSION,</span>
<span class="hl kwa">+    RPMTAG_SUGGESTVERSION,</span>
<span class="hl kwa">+    RPMTAG_ENHANCEVERSION,</span>
<span class="hl kwa">+    RPMTAG_RECOMMENDVERSION,</span>
<span class="hl kwa">+    RPMTAG_SUPPLEMENTVERSION,</span>
     0
 };
 
diff --git a/build/parsePreamble.c b/build/parsePreamble.c
index 5772bef..cbbbee3 100644
<span class="hl kwb">--- a/build/parsePreamble.c</span>
<span class="hl kwa">+++ b/build/parsePreamble.c</span>
&#64;&#64; -821,6 +821,10 &#64;&#64; static rpmRC handlePreambleTag(rpmSpec spec, Package pkg, rpmTagVal tag,
 	}
 	/* fallthrough */
     case RPMTAG_PREREQ:
<span class="hl kwa">+    case RPMTAG_RECOMMENDFLAGS:</span>
<span class="hl kwa">+    case RPMTAG_SUGGESTFLAGS:</span>
<span class="hl kwa">+    case RPMTAG_SUPPLEMENTFLAGS:</span>
<span class="hl kwa">+    case RPMTAG_ENHANCEFLAGS:</span>
     case RPMTAG_CONFLICTFLAGS:
     case RPMTAG_OBSOLETEFLAGS:
     case RPMTAG_PROVIDEFLAGS:
&#64;&#64; -922,6 +926,10 &#64;&#64; static struct PreambleRec_s const preambleList[] = {
     {RPMTAG_ICON,		0, 0, LEN_AND_STR(&quot;icon&quot;)},
     {RPMTAG_PROVIDEFLAGS,	0, 0, LEN_AND_STR(&quot;provides&quot;)},
     {RPMTAG_REQUIREFLAGS,	2, 0, LEN_AND_STR(&quot;requires&quot;)},
<span class="hl kwa">+    {RPMTAG_RECOMMENDFLAGS,	0, 0, LEN_AND_STR(&quot;recommends&quot;)},</span>
<span class="hl kwa">+    {RPMTAG_SUGGESTFLAGS,	0, 0, LEN_AND_STR(&quot;suggests&quot;)},</span>
<span class="hl kwa">+    {RPMTAG_SUPPLEMENTFLAGS,	0, 0, LEN_AND_STR(&quot;supplements&quot;)},</span>
<span class="hl kwa">+    {RPMTAG_ENHANCEFLAGS,	0, 0, LEN_AND_STR(&quot;enhances&quot;)},</span>
     {RPMTAG_PREREQ,		2, 1, LEN_AND_STR(&quot;prereq&quot;)},
     {RPMTAG_CONFLICTFLAGS,	0, 0, LEN_AND_STR(&quot;conflicts&quot;)},
     {RPMTAG_OBSOLETEFLAGS,	0, 0, LEN_AND_STR(&quot;obsoletes&quot;)},
diff --git a/build/parseReqs.c b/build/parseReqs.c
index ba080a1..1427111 100644
<span class="hl kwb">--- a/build/parseReqs.c</span>
<span class="hl kwa">+++ b/build/parseReqs.c</span>
&#64;&#64; -61,6 +61,18 &#64;&#64; rpmRC parseRCPOT(rpmSpec spec, Package pkg, const char *field, rpmTagVal tagN,
 	nametag = RPMTAG_REQUIRENAME;
 	tagflags |= RPMSENSE_ANY;
 	break;
<span class="hl kwa">+    case RPMTAG_RECOMMENDFLAGS:</span>
<span class="hl kwa">+	nametag = RPMTAG_RECOMMENDNAME;</span>
<span class="hl kwa">+	break;</span>
<span class="hl kwa">+    case RPMTAG_SUGGESTFLAGS:</span>
<span class="hl kwa">+	nametag = RPMTAG_SUGGESTNAME;</span>
<span class="hl kwa">+	break;</span>
<span class="hl kwa">+    case RPMTAG_SUPPLEMENTFLAGS:</span>
<span class="hl kwa">+	nametag = RPMTAG_SUPPLEMENTNAME;</span>
<span class="hl kwa">+	break;</span>
<span class="hl kwa">+    case RPMTAG_ENHANCEFLAGS:</span>
<span class="hl kwa">+	nametag = RPMTAG_ENHANCENAME;</span>
<span class="hl kwa">+	break;</span>
     case RPMTAG_PROVIDEFLAGS:
 	nametag = RPMTAG_PROVIDENAME;
 	break;
diff --git a/build/reqprov.c b/build/reqprov.c
index a368f42..c270af6 100644
<span class="hl kwb">--- a/build/reqprov.c</span>
<span class="hl kwa">+++ b/build/reqprov.c</span>
&#64;&#64; -81,6 +81,30 &#64;&#64; int addReqProv(Package pkg, rpmTagVal tagN,
 	extra = Flags &amp; RPMSENSE_TRIGGER;
 	dsp = &amp;pkg-&gt;triggers;
 	break;
<span class="hl kwa">+    case RPMTAG_RECOMMENDNAME:</span>
<span class="hl kwa">+	versiontag = RPMTAG_RECOMMENDVERSION;</span>
<span class="hl kwa">+	flagtag = RPMTAG_RECOMMENDFLAGS;</span>
<span class="hl kwa">+	extra = Flags &amp; _ALL_REQUIRES_MASK;</span>
<span class="hl kwa">+	dsp = &amp;pkg-&gt;recommends;</span>
<span class="hl kwa">+	break;</span>
<span class="hl kwa">+    case RPMTAG_SUGGESTNAME:</span>
<span class="hl kwa">+	versiontag = RPMTAG_SUGGESTVERSION;</span>
<span class="hl kwa">+	flagtag = RPMTAG_SUGGESTFLAGS;</span>
<span class="hl kwa">+	extra = Flags &amp; _ALL_REQUIRES_MASK;</span>
<span class="hl kwa">+	dsp = &amp;pkg-&gt;suggests;</span>
<span class="hl kwa">+	break;</span>
<span class="hl kwa">+    case RPMTAG_SUPPLEMENTNAME:</span>
<span class="hl kwa">+	versiontag = RPMTAG_SUPPLEMENTVERSION;</span>
<span class="hl kwa">+	flagtag = RPMTAG_SUPPLEMENTFLAGS;</span>
<span class="hl kwa">+	extra = Flags &amp; _ALL_REQUIRES_MASK;</span>
<span class="hl kwa">+	dsp = &amp;pkg-&gt;supplements;</span>
<span class="hl kwa">+	break;</span>
<span class="hl kwa">+    case RPMTAG_ENHANCENAME:</span>
<span class="hl kwa">+	versiontag = RPMTAG_ENHANCEVERSION;</span>
<span class="hl kwa">+	flagtag = RPMTAG_ENHANCEFLAGS;</span>
<span class="hl kwa">+	extra = Flags &amp; _ALL_REQUIRES_MASK;</span>
<span class="hl kwa">+	dsp = &amp;pkg-&gt;enhances;</span>
<span class="hl kwa">+	break;</span>
     case RPMTAG_REQUIRENAME:
     default:
 	tagN = RPMTAG_REQUIRENAME;
diff --git a/build/rpmbuild_internal.h b/build/rpmbuild_internal.h
index a9e4c7c..0a1977f 100644
<span class="hl kwb">--- a/build/rpmbuild_internal.h</span>
<span class="hl kwa">+++ b/build/rpmbuild_internal.h</span>
&#64;&#64; -93,6 +93,10 &#64;&#64; struct Package_s {
     rpmds ds;			/*!&lt; Requires: N = EVR */
     rpmds requires;
     rpmds provides;
<span class="hl kwa">+    rpmds recommends;</span>
<span class="hl kwa">+    rpmds suggests;</span>
<span class="hl kwa">+    rpmds supplements;</span>
<span class="hl kwa">+    rpmds enhances;</span>
     rpmds conflicts;
     rpmds obsoletes;
     rpmds triggers;
diff --git a/build/spec.c b/build/spec.c
index 703ec78..7ae2120 100644
<span class="hl kwb">--- a/build/spec.c</span>
<span class="hl kwa">+++ b/build/spec.c</span>
&#64;&#64; -139,6 +139,11 &#64;&#64; static Package freePackage(Package pkg)
     pkg-&gt;ds = rpmdsFree(pkg-&gt;ds);
     pkg-&gt;requires = rpmdsFree(pkg-&gt;requires);
     pkg-&gt;provides = rpmdsFree(pkg-&gt;provides);
<span class="hl kwa">+    pkg-&gt;recommends = rpmdsFree(pkg-&gt;recommends);</span>
<span class="hl kwa">+    pkg-&gt;suggests = rpmdsFree(pkg-&gt;suggests);</span>
<span class="hl kwa">+    pkg-&gt;supplements = rpmdsFree(pkg-&gt;supplements);</span>
<span class="hl kwa">+    pkg-&gt;enhances = rpmdsFree(pkg-&gt;enhances);</span>
<span class="hl kwa">+</span>
     pkg-&gt;conflicts = rpmdsFree(pkg-&gt;conflicts);
     pkg-&gt;obsoletes = rpmdsFree(pkg-&gt;obsoletes);
     pkg-&gt;triggers = rpmdsFree(pkg-&gt;triggers);
diff --git a/lib/rpmds.c b/lib/rpmds.c
index 7a51167..1e5dda0 100644
<span class="hl kwb">--- a/lib/rpmds.c</span>
<span class="hl kwa">+++ b/lib/rpmds.c</span>
&#64;&#64; -52,6 +52,22 &#64;&#64; static int dsType(rpmTagVal tag,
 	t = &quot;Requires&quot;;
 	evr = RPMTAG_REQUIREVERSION;
 	f = RPMTAG_REQUIREFLAGS;
<span class="hl kwa">+    } else if (tag == RPMTAG_SUPPLEMENTNAME) {</span>
<span class="hl kwa">+	t = &quot;Supplements&quot;;</span>
<span class="hl kwa">+	evr = RPMTAG_SUPPLEMENTVERSION;</span>
<span class="hl kwa">+	f = RPMTAG_SUPPLEMENTFLAGS;</span>
<span class="hl kwa">+    } else if (tag == RPMTAG_ENHANCENAME) {</span>
<span class="hl kwa">+	t = &quot;Enhances&quot;;</span>
<span class="hl kwa">+	evr = RPMTAG_ENHANCEVERSION;</span>
<span class="hl kwa">+	f = RPMTAG_ENHANCEFLAGS;</span>
<span class="hl kwa">+    } else if (tag == RPMTAG_RECOMMENDNAME) {</span>
<span class="hl kwa">+	t = &quot;Recommends&quot;;</span>
<span class="hl kwa">+	evr = RPMTAG_RECOMMENDVERSION;</span>
<span class="hl kwa">+	f = RPMTAG_RECOMMENDFLAGS;</span>
<span class="hl kwa">+    } else if (tag == RPMTAG_SUGGESTNAME) {</span>
<span class="hl kwa">+	t = &quot;Suggests&quot;;</span>
<span class="hl kwa">+	evr = RPMTAG_SUGGESTVERSION;</span>
<span class="hl kwa">+	f = RPMTAG_SUGGESTFLAGS;</span>
     } else if (tag == RPMTAG_CONFLICTNAME) {
 	t = &quot;Conflicts&quot;;
 	evr = RPMTAG_CONFLICTVERSION;
diff --git a/lib/rpmtag.h b/lib/rpmtag.h
index 64b03f1..b943229 100644
<span class="hl kwb">--- a/lib/rpmtag.h</span>
<span class="hl kwa">+++ b/lib/rpmtag.h</span>
&#64;&#64; -217,14 +217,14 &#64;&#64; typedef enum rpmTag_e {
     RPMTAG_PRETRANSPROG		= 1153,	/* s[] */
     RPMTAG_POSTTRANSPROG	= 1154,	/* s[] */
     RPMTAG_DISTTAG		= 1155,	/* s */
<span class="hl kwb">-    RPMTAG_SUGGESTSNAME		= 1156,	/* s[] extension (unimplemented) */</span>
<span class="hl kwb">-#define	RPMTAG_SUGGESTS RPMTAG_SUGGESTSNAME	/* s[] (unimplemented) */</span>
<span class="hl kwb">-    RPMTAG_SUGGESTSVERSION	= 1157,	/* s[] extension (unimplemented) */</span>
<span class="hl kwb">-    RPMTAG_SUGGESTSFLAGS	= 1158,	/* i[] extension (unimplemented) */</span>
<span class="hl kwb">-    RPMTAG_ENHANCESNAME		= 1159,	/* s[] extension placeholder (unimplemented) */</span>
<span class="hl kwb">-#define	RPMTAG_ENHANCES RPMTAG_ENHANCESNAME	/* s[] (unimplemented) */</span>
<span class="hl kwb">-    RPMTAG_ENHANCESVERSION	= 1160,	/* s[] extension placeholder (unimplemented) */</span>
<span class="hl kwb">-    RPMTAG_ENHANCESFLAGS	= 1161,	/* i[] extension placeholder (unimplemented) */</span>
<span class="hl kwa">+    RPMTAG_OLDSUGGESTSNAME	= 1156,	/* s[] (unimplemented) */</span>
<span class="hl kwa">+#define	RPMTAG_OLDSUGGESTS RPMTAG_OLDSUGGESTSNAME /* s[] (unimplemented) */</span>
<span class="hl kwa">+    RPMTAG_OLDSUGGESTSVERSION	= 1157,	/* s[] (unimplemented) */</span>
<span class="hl kwa">+    RPMTAG_OLDSUGGESTSFLAGS	= 1158,	/* i[] (unimplemented) */</span>
<span class="hl kwa">+    RPMTAG_OLDENHANCESNAME		= 1159,	/* s[] (unimplemented) */</span>
<span class="hl kwa">+#define	RPMTAG_OLDENHANCES RPMTAG_OLDENHANCESNAME /* s[] (unimplemented) */</span>
<span class="hl kwa">+    RPMTAG_OLDENHANCESVERSION	= 1160,	/* s[] (unimplemented) */</span>
<span class="hl kwa">+    RPMTAG_OLDENHANCESFLAGS	= 1161,	/* i[] (unimplemented) */</span>
     RPMTAG_PRIORITY		= 1162, /* i[] extension placeholder (unimplemented) */
     RPMTAG_CVSID		= 1163, /* s (unimplemented) */
 #define	RPMTAG_SVNID	RPMTAG_CVSID	/* s (unimplemented) */
&#64;&#64; -261,6 +261,7 &#64;&#64; typedef enum rpmTag_e {
     RPMTAG_BUILDOBSOLETES	= 1194, /* internal (unimplemented) */
     RPMTAG_DBINSTANCE		= 1195, /* i extension */
     RPMTAG_NVRA			= 1196, /* s extension */
<span class="hl kwa">+</span>
     /* tags 1997-4999 reserved */
     RPMTAG_FILENAMES		= 5000, /* s[] extension */
     RPMTAG_FILEPROVIDE		= 5001, /* s[] extension */
&#64;&#64; -307,6 +308,26 &#64;&#64; typedef enum rpmTag_e {
     RPMTAG_OBSOLETENEVRS	= 5043, /* s[] extension */
     RPMTAG_CONFLICTNEVRS	= 5044, /* s[] extension */
     RPMTAG_FILENLINKS		= 5045,	/* i[] extension */
<span class="hl kwa">+    RPMTAG_RECOMMENDNAME	= 5046,	/* s[] */</span>
<span class="hl kwa">+#define	RPMTAG_RECOMMENDS RPMTAG_RECOMMENDNAME	/* s[] */</span>
<span class="hl kwa">+    RPMTAG_RECOMMENDVERSION	= 5047,	/* s[] */</span>
<span class="hl kwa">+    RPMTAG_RECOMMENDFLAGS	= 5048,	/* i[] */</span>
<span class="hl kwa">+    RPMTAG_SUGGESTNAME		= 5049,	/* s[] */</span>
<span class="hl kwa">+#define	RPMTAG_SUGGESTS RPMTAG_SUGGESTNAME	/* s[] */</span>
<span class="hl kwa">+    RPMTAG_SUGGESTVERSION	= 5050,	/* s[] extension */</span>
<span class="hl kwa">+    RPMTAG_SUGGESTFLAGS		= 5051,	/* i[] extension */</span>
<span class="hl kwa">+    RPMTAG_SUPPLEMENTNAME	= 5052,	/* s[] */</span>
<span class="hl kwa">+#define	RPMTAG_SUPPLEMENTS RPMTAG_SUPPLEMENTNAME /* s[] */</span>
<span class="hl kwa">+    RPMTAG_SUPPLEMENTVERSION	= 5053,	/* s[] */</span>
<span class="hl kwa">+    RPMTAG_SUPPLEMENTFLAGS	= 5054,	/* i[] */</span>
<span class="hl kwa">+    RPMTAG_ENHANCENAME		= 5055,	/* s[] */</span>
<span class="hl kwa">+#define	RPMTAG_ENHANCES RPMTAG_ENHANCENAME	/* s[] */</span>
<span class="hl kwa">+    RPMTAG_ENHANCEVERSION	= 5056,	/* s[] */</span>
<span class="hl kwa">+    RPMTAG_ENHANCEFLAGS		= 5057,	/* i[] */</span>
<span class="hl kwa">+    RPMTAG_RECOMMENDNEVRS	= 5058, /* s[] extension */</span>
<span class="hl kwa">+    RPMTAG_SUGGESTNEVRS		= 5059, /* s[] extension */</span>
<span class="hl kwa">+    RPMTAG_SUPPLEMENTNEVRS	= 5060, /* s[] extension */</span>
<span class="hl kwa">+    RPMTAG_ENHANCENEVRS		= 5061, /* s[] extension */</span>
 
     RPMTAG_FIRSTFREE_TAG	/*!&lt; internal */
 } rpmTag;
diff --git a/lib/tagexts.c b/lib/tagexts.c
index 29b2bae..e940310 100644
<span class="hl kwb">--- a/lib/tagexts.c</span>
<span class="hl kwa">+++ b/lib/tagexts.c</span>
&#64;&#64; -761,6 +761,26 &#64;&#64; static int requirenevrsTag(Header h, rpmtd td, headerGetFlags hgflags)
     return depnevrsTag(h, td, hgflags, RPMTAG_REQUIRENAME);
 }
 
<span class="hl kwa">+static int recommendnevrsTag(Header h, rpmtd td, headerGetFlags hgflags)</span>
<span class="hl kwa">+{</span>
<span class="hl kwa">+    return depnevrsTag(h, td, hgflags, RPMTAG_RECOMMENDNAME);</span>
<span class="hl kwa">+}</span>
<span class="hl kwa">+</span>
<span class="hl kwa">+static int suggestnevrsTag(Header h, rpmtd td, headerGetFlags hgflags)</span>
<span class="hl kwa">+{</span>
<span class="hl kwa">+    return depnevrsTag(h, td, hgflags, RPMTAG_SUGGESTNAME);</span>
<span class="hl kwa">+}</span>
<span class="hl kwa">+</span>
<span class="hl kwa">+static int supplementnevrsTag(Header h, rpmtd td, headerGetFlags hgflags)</span>
<span class="hl kwa">+{</span>
<span class="hl kwa">+    return depnevrsTag(h, td, hgflags, RPMTAG_SUPPLEMENTNAME);</span>
<span class="hl kwa">+}</span>
<span class="hl kwa">+</span>
<span class="hl kwa">+static int enhancenevrsTag(Header h, rpmtd td, headerGetFlags hgflags)</span>
<span class="hl kwa">+{</span>
<span class="hl kwa">+    return depnevrsTag(h, td, hgflags, RPMTAG_ENHANCENAME);</span>
<span class="hl kwa">+}</span>
<span class="hl kwa">+</span>
 static int providenevrsTag(Header h, rpmtd td, headerGetFlags hgflags)
 {
     return depnevrsTag(h, td, hgflags, RPMTAG_PROVIDENAME);
&#64;&#64; -823,6 +843,10 &#64;&#64; static const struct headerTagFunc_s rpmHeaderTagExtensions[] = {
     { RPMTAG_EPOCHNUM,		epochnumTag },
     { RPMTAG_INSTFILENAMES,	instfilenamesTag },
     { RPMTAG_REQUIRENEVRS,	requirenevrsTag },
<span class="hl kwa">+    { RPMTAG_RECOMMENDNEVRS,	recommendnevrsTag},</span>
<span class="hl kwa">+    { RPMTAG_SUGGESTNEVRS,	suggestnevrsTag},</span>
<span class="hl kwa">+    { RPMTAG_SUPPLEMENTNEVRS,	supplementnevrsTag},</span>
<span class="hl kwa">+    { RPMTAG_ENHANCENEVRS,	enhancenevrsTag},</span>
     { RPMTAG_PROVIDENEVRS,	providenevrsTag },
     { RPMTAG_OBSOLETENEVRS,	obsoletenevrsTag },
     { RPMTAG_CONFLICTNEVRS,	conflictnevrsTag },
diff --git a/rpmpopt.in b/rpmpopt.in
index 805599e..036ab4e 100644
<span class="hl kwb">--- a/rpmpopt.in</span>
<span class="hl kwa">+++ b/rpmpopt.in</span>
&#64;&#64; -67,6 +67,19 &#64;&#64; rpm	alias --requires	--qf \
 	--POPTdesc=$&quot;list capabilities required by package(s)&quot;
 rpm	alias -R --requires
 
<span class="hl kwa">+rpm	alias --recommends	--qf \</span>
<span class="hl kwa">+  &quot;[%|VERBOSE?{%{RECOMMENDFLAGS:deptype}: }:{}|%{RECOMMENDNEVRS}\n]&quot; \</span>
<span class="hl kwa">+	--POPTdesc=$&quot;list capabilities recommended by package(s)&quot;</span>
<span class="hl kwa">+rpm	alias --suggests	--qf \</span>
<span class="hl kwa">+  &quot;[%|VERBOSE?{%{SUGGESTFLAGS:deptype}: }:{}|%{SUGGESTNEVRS}\n]&quot; \</span>
<span class="hl kwa">+	--POPTdesc=$&quot;list capabilities suggested by package(s)&quot;</span>
<span class="hl kwa">+rpm	alias --supplements	--qf \</span>
<span class="hl kwa">+  &quot;[%|VERBOSE?{%{SUPPLEMENTFLAGS:deptype}: }:{}|%{SUPPLEMENTNEVRS}\n]&quot; \</span>
<span class="hl kwa">+	--POPTdesc=$&quot;list capabilities supplemented by package(s)&quot;</span>
<span class="hl kwa">+rpm	alias --enhances	--qf \</span>
<span class="hl kwa">+  &quot;[%|VERBOSE?{%{ENHANCEFLAGS:deptype}: }:{}|%{ENHANCENEVRS}\n]&quot; \</span>
<span class="hl kwa">+	--POPTdesc=$&quot;list capabilities enhanced by package(s)&quot;</span>
<span class="hl kwa">+</span>
 rpm	alias --info --qf '\
 Name        : %{NAME}\n\
 %|EPOCH?{Epoch       : %{EPOCH}\n}|\
diff --git a/tests/data/SPECS/deptest.spec b/tests/data/SPECS/deptest.spec
index cb4cbbd..7c47f6d 100644
<span class="hl kwb">--- a/tests/data/SPECS/deptest.spec</span>
<span class="hl kwa">+++ b/tests/data/SPECS/deptest.spec</span>
&#64;&#64; -10,6 +10,10 &#64;&#64; BuildArch:	noarch
 %{?provs:Provides: %{provs}}
 %{?cfls:Conflicts: %{cfls}}
 %{?obs:Obsoletes: %{obs}}
<span class="hl kwa">+%{?recs:Recommends: %{recs}}</span>
<span class="hl kwa">+%{?sugs:Suggests: %{sugs}}</span>
<span class="hl kwa">+%{?sups:Supplements: %{sups}}</span>
<span class="hl kwa">+%{?ens:Enhances: %{ens}}</span>
 
 %description
 %{summary}
diff --git a/tests/rpmbuild.at b/tests/rpmbuild.at
index 6230903..c4c954c 100644
<span class="hl kwb">--- a/tests/rpmbuild.at</span>
<span class="hl kwa">+++ b/tests/rpmbuild.at</span>
&#64;&#64; -185,3 +185,25 &#64;&#64; lrwxrwxrwx /opt/globtest/linkgood
 ],
 [])
 AT_CLEANUP
<span class="hl kwa">+</span>
<span class="hl kwa">+# ------------------------------</span>
<span class="hl kwa">+# Check if weak and reverse requires can be built</span>
<span class="hl kwa">+AT_SETUP([Weak and reverse requires])</span>
<span class="hl kwa">+AT_KEYWORDS([build])</span>
<span class="hl kwa">+AT_CHECK([</span>
<span class="hl kwa">+</span>
<span class="hl kwa">+runroot rpmbuild -bb --quiet \</span>
<span class="hl kwa">+	--define &quot;pkg weakdeps&quot; \</span>
<span class="hl kwa">+	--define &quot;recs foo &gt; 1.2.3&quot; \</span>
<span class="hl kwa">+	--define &quot;sugs bar &gt;= 0.1.2&quot; \</span>
<span class="hl kwa">+	--define &quot;sups baz&quot; \</span>
<span class="hl kwa">+	--define &quot;ens zap = 3&quot; \</span>
<span class="hl kwa">+	  /data/SPECS/deptest.spec</span>
<span class="hl kwa">+</span>
<span class="hl kwa">+runroot rpm -qp --qf &quot;[%{supplementname}\n]&quot; /build/RPMS/noarch/deptest-weakdeps-1.0-1.noarch.rpm</span>
<span class="hl kwa">+],</span>
<span class="hl kwa">+[0],</span>
<span class="hl kwa">+[baz</span>
<span class="hl kwa">+],</span>
<span class="hl kwa">+[ignore])</span>
<span class="hl kwa">+AT_CLEANUP</span>
diff --git a/tests/rpmgeneral.at b/tests/rpmgeneral.at
index 13131e2..80cca63 100644
<span class="hl kwb">--- a/tests/rpmgeneral.at</span>
<span class="hl kwa">+++ b/tests/rpmgeneral.at</span>
&#64;&#64; -79,6 +79,11 &#64;&#64; DISTTAG
 DISTURL
 DSAHEADER
 E
<span class="hl kwa">+ENHANCEFLAGS</span>
<span class="hl kwa">+ENHANCENAME</span>
<span class="hl kwa">+ENHANCENEVRS</span>
<span class="hl kwa">+ENHANCES</span>
<span class="hl kwa">+ENHANCEVERSION</span>
 EPOCH
 EPOCHNUM
 EVR
&#64;&#64; -199,6 +204,11 &#64;&#64; PROVIDES
 PROVIDEVERSION
 PUBKEYS
 R
<span class="hl kwa">+RECOMMENDFLAGS</span>
<span class="hl kwa">+RECOMMENDNAME</span>
<span class="hl kwa">+RECOMMENDNEVRS</span>
<span class="hl kwa">+RECOMMENDS</span>
<span class="hl kwa">+RECOMMENDVERSION</span>
 RECONTEXTS
 RELEASE
 REMOVETID
&#64;&#64; -219,7 +229,17 &#64;&#64; SOURCE
 SOURCEPACKAGE
 SOURCEPKGID
 SOURCERPM
<span class="hl kwa">+SUGGESTFLAGS</span>
<span class="hl kwa">+SUGGESTNAME</span>
<span class="hl kwa">+SUGGESTNEVRS</span>
<span class="hl kwa">+SUGGESTS</span>
<span class="hl kwa">+SUGGESTVERSION</span>
 SUMMARY
<span class="hl kwa">+SUPPLEMENTFLAGS</span>
<span class="hl kwa">+SUPPLEMENTNAME</span>
<span class="hl kwa">+SUPPLEMENTNEVRS</span>
<span class="hl kwa">+SUPPLEMENTS</span>
<span class="hl kwa">+SUPPLEMENTVERSION</span>
 TRIGGERCONDS
 TRIGGERFLAGS
 TRIGGERINDEX
</code></pre></td></tr></table>
</div> <!-- class=content -->
<div class='footer'>generated  by cgit v0.9.2-21-gd62e at 2015-01-26 10:45:58 (GMT)</div>
</div> <!-- id=cgit -->
</body>
</html>
